integrations/docker/images/chip-build-android/build.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
this should be a symbolic link up 2 directories

Reply:
done